apiserver/utility/utility.go

```diff
@@ -27,61 +26,31 @@ func ValidateBitriseConfigAndSecret(bitriseConfig, secretsConfig string) (*Warni
 	bitriseConfigBase64 := base64.StdEncoding.EncodeToString([]byte(bitriseConfig))
 	secretsConfigBase64 := base64.StdEncoding.EncodeToString([]byte(secretsConfig))
 
-	validateCmd := command.New("bitrise",
-		"-l=panic", "validate", "--format=json",
-		"--config-base64", bitriseConfigBase64,
-		"--inventory-base64", secretsConfigBase64)
-
-	combinedOut, cmdErr := validateCmd.RunAndReturnTrimmedCombinedOutput()
-
-	type BitriseCLIValidateItemModel struct {
-		IsValid  bool     `json:"is_valid"`
-		Error    string   `json:"error"`
-		Warnings []string `json:"warnings"`
-	}
-	type BitriseCLIValidateDataModel struct {
-		Config  BitriseCLIValidateItemModel `json:"config"`
-		Secrets BitriseCLIValidateItemModel `json:"secrets"`
-	}
-	type BitriseCLIValidateOutputModel struct {
-		Data  BitriseCLIValidateDataModel `json:"data"`
-		Error string                      `json:"error"`
-	}
-
-	var validationOutput BitriseCLIValidateOutputModel
-	if outputParseErr := json.NewDecoder(strings.NewReader(combinedOut)).Decode(&validationOutput); outputParseErr != nil {
-		if cmdErr != nil {
-			return nil, fmt.Errorf("Failed to run bitrise validate command, error: %s | 'bitrise validate' command output: %s", cmdErr, combinedOut)
-		}
-		return nil, fmt.Errorf("Failed to parse bitrise validate output, error: %s | 'bitrise validate' command output: %s", outputParseErr, combinedOut)
-	}
+	_, bitriseWarns, bitriseErr := cli.CreateBitriseConfigFromCLIParams(bitriseConfigBase64, "")
+	_, secretsErr := cli.CreateInventoryFromCLIParams(secretsConfigBase64, "")
```

Comment:
this should be only called if the secrets base 64 is not empty otherwise, it can pick up the secrets file in the current working directory, which should not happen: https://github.com/bitrise-io/bitrise/blob/518e23eca82dc826ef7ee1e27e15878dd8118bb3/cli/run_util.go#L252